src/Manager/Environment/environmentEffects.js

```diff
@@ -201,6 +235,19 @@ const unpack = (version, src, dest) => async dispatch => {
         }
         default:
     }
+
+    const unpackInfo = `${version}; ${process.platform}; ${process.arch}`;
+    sendUsageData(EventAction.UNPACK_TOOLCHAIN_SUCCESS, unpackInfo);
+    sendUsageData(
+        EventAction.UNPACK_TOOLCHAIN_TIME,
+        `${calculateTimeConsumed(unpackTimeStart)} min; ${unpackInfo}`
+    );
+    logger.info(
+        `Finish unpacking toolchain ${unpackInfo} in approximate ${calculateTimeConsumed(
```

Comment:
[minor] I suggest the wording here should be something similar to what I suggested above.

Reply:
Since the message shows that the process has completed, I will update the sentence like this.
"Version ${version} of the toolchain has finished downloading in approximately ${timeConsumed} minutes"
It is a correct sentence?

Reply:
Almost, just remove "has", i.e. "Version ${version} of the toolchain finished downloading in approximately ${timeConsumed} minutes".

Maybe we can remove "approximately" though since we know how long it took? Could do something like this:
"Finished downloading version ${version} of the toolchain after ${timeConsumed} minutes."

Reply:
We can provide a precise time consumed. The reason for using approximate minutes is that it's easier to see the distribution on GA. If I use a precise time, then each time will be an individual event, instead of a group. We will need extra effort to process the data to group them to see the distribution.